src/components/Header/NavTitle.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import Dropdown from "./dropdown";
+
+function NavTitle(props) {
+  const [focus, setFocus] = useState(false);
+  const [click, setClick] = useState(false);
+  const [dimensions, setDimensions] = useState({
+    height: window.innerHeight,
+    width: window.innerWidth,
+  });
+
+  useEffect(() => {
+    function handleResize() {
+      setDimensions({
+        height: window.innerHeight,
+        width: window.innerWidth,
+      });
+    }
+
+    if (dimensions.width > 900) setClick(false);
+    window.addEventListener("resize", handleResize);
```

Comment:
Yeah I see this this is a tricky case, but it was implemented same in the example app
It's fine for now, just keep in mind you can find better solutions for such dropdown features and don't use such patterns in production :)